internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -788,24 +801,32 @@ func (t *Translator) buildLoadBalancer(policy *egv1a1.BackendTrafficPolicy) *ir.
 		}
 	}
 
-	return lb
+	return lb, nil
 }
 
-func (t *Translator) buildConsistentHashLoadBalancer(policy *egv1a1.BackendTrafficPolicy) *ir.ConsistentHash {
+func (t *Translator) buildConsistentHashLoadBalancer(policy *egv1a1.BackendTrafficPolicy) (*ir.ConsistentHash, error) {
```

Comment:
can we directly pass `policy.Spec.LoadBalancer.ConsistentHash`, in instead of the whole `BackendTrafficPolicy`?

Reply:
The reason I pass the whole `BackendTrafficPolicy` is many existing internal building functions in function `buildAbc(policy)` used to building `Spec.Abc` is invoked as `buildAbcXyz(policy)`.

To keep the same function signature style.

- `buildHealthCheck(policy)`
  - `buildPassiveHealthCheck(policy)`
  - `buildActiveHealthCheck(policy)`
- `buildRateLimit(policy)`
  - `buildGlobalRateLimit(policy)`
  - `buildLocalRateLimit(policy)`

Reply:
got it! lgtm anyway